src/CoreBundle/private/library/classes/TCMSFields/TCMSFieldDownloads.class.php

```diff
@@ -60,14 +60,20 @@ protected function GetAttachedDocumentListAsHTML($bReadOnly = false)
       <div class=\"pt-2\">\n";
         while ($oDownload = &$oDownloads->Next()) {
             /** @var $oDownload TCMSDownloadFile */
+
+            $tdWidth = 50;
+            if (true === $bReadOnly) {
```

Comment:
Same question here: -> Twig?

Especially if this "readonly" is only display logic.

Reply:
Nope. Would be a big change that has nothing to do with the icons.